### ISSUE TITLE
Cannot set property 'more_btn' of undefined on Froala V4

### ISSUE THREAD

**Issue by carla-at-wiris** [froala/wysiwyg-editor#4323]
##### Expected behavior.
No console Error

##### Actual behavior.
A console error appears:
```
Uncaught TypeError: Cannot set property 'more_btn' of undefined     froala_editor.pkgd.min.js:7
    at r (froala_editor.pkgd.min.js:7)
    at froala_editor.pkgd.min.js:7
```

##### Steps to reproduce the problem.

1. Create a Froala integration from scratch following [this](https://froala.com/wysiwyg-editor/docs/overview/) steps.
2. Open the created demo.
3. See the console error.

Even though there's an error, we haven't experienced any breaks on our application, and every time we refresh the page, the error disappears, it appears again once we open a fresh page.

Our .js code:
```
import 'froala-editor/css/froala_editor.pkgd.min.css';
new FroalaEditor('#editor', { 
  // Define the toolbar options for the froala editor.
  toolbarButtons: ['undo', 'redo', 'bold', 'italic', '|', 'insertImage'],
});
```
We load the `froala_editor.pkgd.min.js` in our .html file.
##### Editor version.
4.0.4

##### OS.
Ubuntu

##### Browser.
Chrome (on Firefox also appears an error but is a bit different: `Uncaught TypeError: r is undefined`)

##### Recording.

https://user-images.githubusercontent.com/60604864/131879069-109a5423-72d0-41f8-b603-2aad7faa652f.mp4

**Comment by awegmueller** [comment]
Same error here with version 4.0.4. It seems like the editor assumes the track-changes commands are always available. But that's not the case. 

The error occurs in the _showScreenButtons() function when a user resizes the browser window on this line, which returns undefined:
`var cmdInfo = FroalaEditor.COMMANDS[cmdName];`

A simple patch which works for us:

```
function _showScreenButtons() {
  // <BSI Patch> fix TypeError on resize, which occurs because trackingBtns are not valid COMMANDS
  if (editor.opts.trackChangesEnabled && $("[data-name='trackChanges-".concat(editor.id, "']")).length == 0) {
  ...
```

**Comment by ilyaskarim** [comment]
Fixed in latest version 4.0.5

**Comment by carla-at-wiris** [comment]
@ilyaskarim, Thank you so much!
